lib/widgets/message_list.dart

```diff
@@ -496,14 +496,14 @@ class _UnreadMarker extends StatelessWidget {
             // See zulip:web/styles/message_row.css .
             duration: Duration(milliseconds: isRead ? 2000 : 300),
             curve: Curves.easeOut,
-            child: DecoratedBox(
+            child: const DecoratedBox(
               decoration: BoxDecoration(
                 color: color,
                 // TODO(#95): Don't show this extra border in dark mode, see:
                 //   https://github.com/zulip/zulip-flutter/pull/317#issuecomment-1784311663
                 border: Border(left: BorderSide(
                   width: 1,
-                  color: Colors.white.withOpacity(0.6))))))),
+                  color: Color(0x99ffffff))))))),
```

Comment:
The `withOpacity` computation is pretty trivial, unlike `HSLColor.toColor`. So this can just remain as it is.